lib/ClangImporter/ClangImporter.cpp

```diff
@@ -4215,9 +4207,22 @@ ClangImporter::getSwiftExplicitModuleDirectCC1Args() const {
 
   if (!Impl.SwiftContext.SearchPathOpts.ScannerPrefixMapper.empty()) {
     // Remap all the paths if requested.
+    // FIXME: Should we also do
+    // https://github.com/swiftlang/llvm-project/pull/10723?
```

Comment:
Can we find this out rather than add a fixme? Or add an issue and point to that instead. Otherwise this will just sit here forever more 😅